aboutcitus/concepts.rst

```diff
@@ -0,0 +1,125 @@
+
+Nodes: Coordinator and Workers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Every cluster has one special node called the *coordinator* (the others are known as workers). Applications send their queries to the coordinator node which relays it to the relevant workers and accumulates the results.
+
+For each query, the coordinator either *routes* it to a single worker node, or *parallelizes* it across several depending on whether the required data lives on a single node or multiple.  The coordinator knows how to do this by consulting its metadata tables. These Citus-specific tables track the DNS names and health of worker nodes, and the distribution of data across nodes. For more information, see our :ref:`metadata_tables`.
+
+Citus also supports an :ref:`mx` mode to allow queries directly against workers. This allows both reads and writes to be scaled even more across all nodes.
+
+Table Types
+-----------
+
+There are three types of tables in a Citus cluster, each used for different purposes.
+
+Type 1: Distributed Tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first type, and most common, is *distributed* tables. These appear to be normal tables to SQL statements, but are horizontally *partitioned* across worker nodes.
+
+.. image:: ../images/diagram-parallel-select.png
+
+Here the rows of ``table`` are stored in tables ``table_1001``, ``table_1002`` etc on the workers. The component worker tables are called *shards*.
+
+Citus runs not only SQL but DDL statements throughout a cluster, so changing the schema of a distributed table cascades to update all the table's shards across workers. See :ref:`ddl`.
+
+Distribution Column
+!!!!!!!!!!!!!!!!!!!
+
+Citus uses algorithmic sharding to assign rows to shards. This means the assignment is made deterministically -- in our case based on the value of a particular table column called the *distribution column.* The cluster administrator must designate this column when distributing a table. Making the right choice is important for performance and functionality, as described in the general topic of :ref:`Distributed Data Modeling <distributed_data_modeling>`.
+
+Type 2: Reference Tables
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+A reference table is a type of distributed table whose entire contents are concentrated into a single shard which is replicated on every worker. Thus queries on any worker can access the reference information locally, without the network overhead of requesting rows from another node. Reference tables have no distribution column because there is no need to distinguish separate shards per row.
```

Comment:
Can we link to the reference tables section here? I'd like for us to cross-link to more of our existing content which dives deeper into these concepts. That hopefully should help with our bounce rate. It also allows people to keep learning more.